app/sass/_casestudy.scss

```diff
@@ -0,0 +1,111 @@
+$highlight:#5FC3AD;
+$dark-grey:#4A4A4A;
+$med-grey:#6D6E70;
+$light-grey:#979797;
+
+._lo-customers{
+    font-family: helvetica;
+    margin: 0px 300px;
+
+    .pg{
+        overflow: visible;
+        &:before,
+        &:after {
+          content: " ";
+          display: table;
+        }
+        &:after {
+          clear:both;
+        }
+    }
+
+  h1{
+      color: $med-grey;
+      text-align: center;
+      font-size: 1.875em;
+      font-weight: 500;
+      margin-top: 2em;
+      margin-bottom: 0em;
+  }
+
+  h2{
+      color: $light-grey;
+      text-align: center;
+      font-size: 1.35em;
+      font-weight: 500;
+      margin: 50px 0 10px;
+  }
+
+  h3 {
+      color: $highlight;
+      font-size: 1.250em;
+      font-weight: 400;
+      margin-top: 10px;
+      margin-bottom: 5px;
+  }
+
+  h4 {
+      color: $med-grey;
+      font-size: 1em;
+      font-weight: 500;
+      margin-top: 24px;
```

Comment:
When using `24px`, maybe use the sass variable for that: `$vs`. See the [_vars.scss](https://github.com/getroadmap/roadmap-site/blob/master/app/sass/_vars.scss#L12) file. Right now, its 1.5*16px.